addons/udes_stock/tests/test_picking_batch.py

```diff
@@ -1705,3 +1705,165 @@ def test04_rereservation_for_same_batch(self):
 
         batch.reserve_pallet('UDES11111')
         batch.reserve_pallet('UDES11111')
+
+
+class TestMultipleOrders(common.BaseUDES):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.pack_4apples_info = [{'product': cls.apple,
+                                  'qty': 4}]
+        cls.pack_4bananas_info = [{'product': cls.banana,
+                                  'qty': 4}]
+        # enable unpickable items by default
+        cls.picking_type_pick.u_enable_unpickable_items = True
+
+        cls.picking_type_pick.u_reserve_pallet_per_picking = True
+
+    def setUp(self):
+        super().setUp()
+        Package = self.env['stock.quant.package']
+
+        self.package_one = Package.get_package("test_package_one", create=True)
```

Comment:
Do we need these packages?

Reply:
No, they're copy-pasta from the original reservation tests - the test code needs some clean up.